GameData/RealismOverhaul/RO_SuggestedMods/Buran-Energia/ENERGIA.cfg

```diff
@@ -0,0 +1,1722 @@
+	%manufacturer = RKK Energia
+	@attachRules = 1,1,1,1,0
+	@fuelCrossFeed = False	
+	!RESOURCE[LiquidFuel]
+	{
+	}
+	!RESOURCE[Oxidizer]
+	{
+	}
+	MODULE
+	{
+		name = ModuleFuelTanks
+		volume = 0
+		type = Default
+		basemass = -1
+		TANK
+		{
+			name = Kerosene
+			amount = 0
+			maxAmount = 0
+		}
+		TANK
+		{
+			name = LqdOxygen
+			amount = 0
+			maxAmount = 0
+		}
+	}
+}
+
+
+@PART[Energia_Left_Power_Units]:FOR[RealismOverhaul]
+{
+	%RSSROConfig = True
+	@MODEL
+	{
+		@scale = 1.0, 1.0, 1.0
+	}
+	%scale = 1.0
+	@rescaleFactor = 1.5625
+	@mass = 0.23665
+	@maxTemp = 1500
+	%manufacturer = RKK Energia
+	@attachRules = 1,1,1,1,0
+	@fuelCrossFeed = False	
+	!RESOURCE[LiquidFuel]
+	{
+	}
+	!RESOURCE[Oxidizer]
+	{
+	}
+	MODULE
+	{
+		name = ModuleFuelTanks
+		volume = 0
+		type = Default
+		basemass = -1
+		TANK
+		{
+			name = Kerosene
+			amount = 0
+			maxAmount = 0
+		}
+		TANK
+		{
+			name = LqdOxygen
+			amount = 0
+			maxAmount = 0
+		}
+	}
+}
+
+
+@PART[BURAN_T]:FOR[RealismOverhaul]
+{
+	%RSSROConfig = True
+	@MODEL
+	{
+		@scale = 1.0, 1.0, 1.0
+	}
+	%scale = 1.0
+	@rescaleFactor = 1.5625
+	@mass = 8.03615           //10.03615
+	@maxTemp = 1500
+	%manufacturer = RKK Energia
+	@attachRules = 1,0,1,0,1
+	@fuelCrossFeed = False	
+	!RESOURCE[LiquidFuel]
+	{
+	}
+	!RESOURCE[Oxidizer]
+	{
+	}
+	MODULE
+	{
+		name = ModuleFuelTanks
+		volume = 0
+		type = Default
+		basemass = -1
+		TANK
+		{
+			name = Kerosene
+			amount = 0
+			maxAmount = 0
+		}
+		TANK
+		{
+			name = LqdOxygen
+			amount = 0
+			maxAmount = 0
+		}
+	}
+}
+
+
+
+
+
+
+
+@PART[Block_Z]:FOR[RealismOverhaul]
+{
+	%RSSROConfig = True
+	@MODEL
+	{
+		@scale = 1.0, 1.0, 1.0
+	}
+	%scale = 1.0
+	@rescaleFactor = 1.5625
+	@mass = 150.00665
+	@maxTemp = 94500
+	%manufacturer = RKK Energia
+	@attachRules = 1,0,1,0,1
+	@fuelCrossFeed = False	
+	!RESOURCE[LiquidFuel]
+	{
+	}
+	!RESOURCE[Oxidizer]
+	{
+	}
+	MODULE
+	{
+		name = ModuleFuelTanks
+		volume = 0
+		type = Default
+		basemass = -1
+		TANK
+		{
+			name = Kerosene
+			amount = 0
+			maxAmount = 0
+		}
+		TANK
+		{
+			name = LqdOxygen
+			amount = 0
+			maxAmount = 0
+		}
+	}
+}
+
+@PART[LAUNCH_PAD]:FOR[RealismOverhaul]
+{
+	%RSSROConfig = True
+	@MODEL
+	{
+		@scale = 1.0, 1.0, 1.0
+	}
+	%scale = 1.0
+	@rescaleFactor = 1.5625
+	@mass = 1150.00665
+	@maxTemp = 98500
+	%manufacturer = RKK Energia
+	@attachRules = 1,0,1,1,0
+	@fuelCrossFeed = False	
+	!RESOURCE[LiquidFuel]
+	{
+	}
+	!RESOURCE[Oxidizer]
+	{
+	}
+	MODULE
+	{
+		name = ModuleFuelTanks
+		volume = 0
+		type = Default
+		basemass = -1
+		TANK
+		{
+			name = Kerosene
+			amount = 0
+			maxAmount = 0
+		}
+		TANK
+		{
+			name = LqdOxygen
+			amount = 0
+			maxAmount = 0
+		}
+	}
+}
+
+
+//Запас топливного бака
+//керосин бак		776,2 т
+//кислород бак		886,8т
+
+@PART[RD170ENERGIA]:FOR[RealismOverhaul]
+{
+    %RSSROConfig = True
+
+    !mesh = NULL
+
+    @MODEL
+    {
+		@model = RealEngines/RD170ENERGIA
+        @scale = 1.0, 1.0, 1.0
+    }
+

```

Comment:
RO does not depend on RealEngines.  What's the point of configuring for Buran and then replacing its model anyway?